woocommerce/woocommerce
# Send emails for cancelled_to_* orders

### All Submissions:






### Changes proposed in this Pull Request:



Closes #20991.

This PR adds email triggers for cancelled_to_on-hold, cancelled_to_processing, and cancelled_to_completed order status changes. This will give the same email behavior for `cancelled_to_*` status transitions that we have now for `failed_to_*` status transitions.

### How to test the changes in this Pull Request:

1. Open up mailcatcher.
2. In admin, set an order status to cancelled, then to processing/on-hold/completed.
3. Verify emails get sent on those status transitions.

### Other information:




### Changelog entry

> Tweak - Send emails when order goes from cancelled status to a paid status.
